launchable/commands/subset.py

```diff
@@ -117,14 +117,26 @@ def separator(self, s: str):
             self._separator = s
 
         def test_path(self, path: TestPathLike):
+            def rel_base_path(path: str):
+                p = path
+                if base_path:
+                    p = normpath(relpath(path, start=base_path))
+
+                return p
+
             if isinstance(path, str) and any(s in path for s in ('*', "?")):
                 for i in glob.iglob(path, recursive=True):
                     if os.path.isfile(i):
-                        self.test_paths.append(self.to_test_path(i))
+                        self.test_paths.append(
+                            self.to_test_path(rel_base_path(i)))
                 return
 
             """register one test"""
-            self.test_paths.append(self.to_test_path(path))
+            p = path
+            if base_path and isinstance(p, str):
```

Comment:
Why did you check `isinstance(p, str)` here, but not in the `rel_base_path()` function? If you needed it in `rel_base_path()`, you could integrate it into the function.

Reply:
Sounds nice, I'll fix it.